r/R/worker_helpers.R

```diff
@@ -135,3 +133,23 @@ handlePagination <- function(gene_results, offset, limit, order_by, order_decrea
   gene_results <- na.omit(gene_results[(offset):(offset + limit), ])
   return(list(gene_results = gene_results, full_count = full_count))
 }
+
+insert_set_child_through_api <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
```

Comment:
as much as it pains me to do this

```suggestion
insertSetChildThroughApi <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
```